src/encoded/static/components/static-pages/placeholders/PageCarousel.js

```diff
@@ -0,0 +1,215 @@
+                'badgeBgColor': null,
+                'link': '/visualization/index'
+            },
+            {
+                'img': 'https://s3.amazonaws.com/4dn-dcic-public/static-pages/carousel-images/heterogeneitypublicationfigures.png',
+                'description': 'Finn et. al. utilize high throughput imaging to study heterogeneity in 3D genome structure.',
+                'title': 'Data Highlight',
+                'badge': null,
+                'badgeBgColor': null,
+                'link': '/publications/80007b23-7748-4492-9e49-c38400acbe60/'
+            }
+        ],
+        'slidesToShow': 3, //integer: Number of slides to show at once
+        'slidesToScroll': 1, //integer: Slides to scroll at once
+        'slideIndex': 0, //integer: the index of the slide to be shown
+        'autoPlay': false, //bool: starts slide show automatically. not working yet due to a bug.
+        'pauseOnHover': true,
+        'wrapAround': true,
+        'dragging': false,
+        'navControlPosition': 'outside', //left and right nav controls position. Possible values: 'outside' or 'inside'
+        'slideHeight': 240, //integer: Slide height in px. If not defined default is 240.
+        //'adjustImageHeight': true, //boolean: if true, resizes slide image's height to container. best for single display power point slides, otherwise set it as false (recommended)
+        'showSlideCount': true, //boolean: displays [Slide x of n] at the bottom left corner. Valid if slidesToShow is defined as 1
+        //'textPosition': 'bottom', //'bottom/top/left/right': position of title & description on slide
+    };
+
+    static propTypes = {
+        'slides': PropTypes.array.isRequired,
+        'slidesToShow': PropTypes.number.isRequired,
+        'slidesToScroll': PropTypes.number.isRequired,
+        'slideIndex': PropTypes.number.isRequired,
+        'autoPlay': PropTypes.bool.isRequired,
+        'pauseOnHover': PropTypes.bool.isRequired,
+        'dragging': PropTypes.bool.isRequired,
+        'windowWidth': PropTypes.number,
+        'navControlPosition': PropTypes.string,
+        'slideHeight': PropTypes.number.isRequired,
+        //'adjustImageHeight': PropTypes.bool.isRequired,
+        'showSlideCount': PropTypes.bool.isRequired,
+        //'textPosition': PropTypes.bool.isRequired,
+    };
+
+    constructor(props){
+        super(props);
+        this.renderSlide = this.renderSlide.bind(this);
+    }
+
+    static refFunc(elem) {
+        setTimeout(() => {
+            if (!elem) return;
+            elem.style.opacity = 1;
+        }, 10);
+    }
+
+    /**
+     * this function renders a slide and customize it based on title/description/badge/link and image
+     * @param {Object} slide    Carousel slide to be displayed
+     */
+    renderSlide(slide) {
+        const { img, description, title, badge, badgeBgColor, link } = slide;
+        const { slideHeight, adjustImageHeight, textPosition } = this.props;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['textPosition' is missing in props validation (react/prop-types)](https://app.codacy.com/app/4dn/fourfront/pullRequest?prid=4476879)